src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Checkbox } from './index';
+import pkg from './package.json';
+import { buildExcludeArgTypes } from '../../storybook/utils';
+import Button from '../Button';
+
+export default {
+  title: 'Components/Checkbox',
+  component: Checkbox,
+  parameters: {
+    pkg,
+  },
+  argTypes: buildExcludeArgTypes(['variation', 'className']),
+} as ComponentMeta<typeof Checkbox>;
+
+const Template: ComponentStory<typeof Checkbox> = (args) => {
+  const onSubmit = (e: React.BaseSyntheticEvent) => {
+    e.preventDefault();
+
+    console.log('----------------------------------------');
+    console.log(' stories onSubmit:');
+    const formData = new FormData(document.querySelector('form') as HTMLFormElement);
+    for (const pair of formData.entries()) {
+      console.log('   ', pair[0] + ': ' + pair[1]);
```

Comment:
remove console logs?

Reply:
I left the logs on purpose to see the results in console (how it behaves in a form).

Reply:
Is the form behavior something we really want to verify in storybook? IMO its more important to keep the stories simple, we can verify the form behavior when (and if) we have an actual form component to show. But it's up to you, won't block on this